sql/core/src/main/scala/org/apache/spark/sql/expressions/WindowSpec.scala

```diff
@@ -174,28 +191,22 @@ class WindowSpec private[sql](
    */
   // Note: when updating the doc for this method, also update Window.rangeBetween.
   def rangeBetween(start: Long, end: Long): WindowSpec = {
```

Comment:
Maybe we can use `Literal(0)` to represent `CurrentRow`? And a sufficient large number(like `Literal(Long.MaxValue)`) for `Unbounded`?